controls/V-92975.rb

```diff
@@ -153,7 +153,7 @@
           date_difference = expiration_date.mjd - password_date.mjd
           describe "Account expiration set for #{user_name}" do
             subject { date_difference }
-            it { should cmp <= input('temporary_account_period') }
+            it { should cmp <= input('temporary_account_period')/24 }
```

Comment:
Should we wrap this in a floor - [geeksforgeeks.org/ruby-integer-floor-function-with-example](https://www.geeksforgeeks.org/ruby-integer-floor-function-with-example/)